app/validators/word_count_validator.rb

```diff
@@ -0,0 +1,9 @@
+class WordCountValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if value.blank?
+
+    if value.scan(/\S+/).size > options[:maximum]
```

Comment:
Include a comment about this needing to match the design system pattern/JS logic there?